### PR TITLE
[NEWSTACK-133] PSO 6.2.0 release

### PR DESCRIPTION
**Pull request by pure-jliao**

### CODE REVIEW

File: docs/flasharray-nvmeof.md [hunk 1]
```diff
@@ -20,6 +21,7 @@ starting from PSO v6.2.0, explicitly NVMeoF-RoCE.
 
 * Follow our [knowledge base](https://support.purestorage.com/Solutions/Linux/Procedures/NVMe//RoCE_Initiator_Setup_for_RHEL//CentOS_7.6)
 to setup and configure your hosts to use NVMe first (skip persistent settings). PSO doesn't configure the environment automatically.
+* For Ubuntu users, please note native NVMe multipath has to be disabled to use this feature. Run `cat /proc/cmdline` and you should see `nvme_core.multipath=N` in the output.
```
Comment (Pure-AdamuKaapan) [inline docs/flasharray-nvmeof.md:24]:
```suggestion
* For Ubuntu users, please note native NVMe multipath has to be disabled to use this feature. Run `cat /proc/cmdline` and you should see `nvme_core.multipath=N` in the output to indicate that it is disabled.
```

**Review by Pure-AdamuKaapan — APPROVED** [review]
LGTM other than the one little sentence nit!

**Review by sdodsley — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+* FlashArray NVMeoF-RoCE support (NVMeoF using RDMA over Converged Ethernet) officially goes GA
+* Enable FlashBlade filesystem hard limit by default upon creation (existing filesystems will not be affected)
+
+#### Bug fixes and enhancements:
+
+* Bug fix for using FC SAN type on FlashArray Purity 6.0+
+* Bug fix for DB reconstruction tool when found devices with no UUIDs
+* NVMeof-RoCE support bug fixes and improvements
+* Topology feature performance improvement
+* Bug fix for iSCSI CIDR not passed to pso-db pods
```
Comment (dsupure) [inline CHANGELOG.md:16]: Can you add the github issue link here?

**Pull request merged**